test/user.test.js

```diff
@@ -0,0 +1,48 @@
+const mongoose = require('mongoose');
+const { graphql } = require('graphql');
+const { makeExecutableSchema } = require('graphql-tools');
+import typeDefs from '../graphql/typeDefs';
+import resolvers from '../graphql/resolvers';
+const graphqlSchema = makeExecutableSchema({
```

Comment:
Should we export the schema from `./handler.js` for use here so we have consistency across the testing and production/staging graphql clients?

Reply:
Absolutely. Was thinking that this is not very dry but wanted to test the mood first. I'll do that tomorrow.